blockchain/types/block.go

```diff
@@ -117,6 +117,10 @@ func (h *Header) Size() common.StorageSize {
 	return common.StorageSize(unsafe.Sizeof(*h)) + common.StorageSize(len(h.Extra)+(h.BlockScore.BitLen()+h.Number.BitLen()+h.Time.BitLen())/8)
 }
 
+func (h *Header) Round() byte {
```

Comment:
how about changing the return type to int64?

Reply:
Origin data is byte. So It doesn't support bigger type than byte. So I used it. :)